runtime-decompiler/src/main/java/org/jrd/backend/data/Help.java

```diff
@@ -270,7 +271,7 @@ String manFormat(String line) {
 
         @Override
         public void printTitle() {
-            System.out.println(".TH JRD 1 \"August 2020\"");
+            System.out.println(".TH JRD 1 \"August 2021\"");
```

Comment:
what is this?

it should be 5, and should be whatever, just not hrdcoded. best to read it from pom.

Reply:
Is it printed also in -h? Should be not...  But I guess it was already there

Reply:
Should be print in -version  I guess there is noting like that :D

Reply:
Also the date shouldeb date of build. Easy for man, worse for cli.

Reply:
Sorry. Has pushed commit propagating build info into jars. Pls use it for this.

Reply:
> Is it printed also in -h? Should be not... But I guess it was already there

No it's not, CLI formatter has printTitle() method empty.